lib/wallet/src/Cardano/Wallet/Write/Tx.hs

```diff
@@ -116,9 +118,13 @@ module Cardano.Wallet.Write.Tx
     , utxoFromTxOutsInRecentEra
     , utxoFromTxOutsInLatestEra
     , utxoFromTxOuts
+    , fromCardanoTx
     , toCardanoUTxO
     , fromCardanoUTxO
+    , toCardanoValue
 
+    -- * Balancing
+    , evaluateTransactionBalance
```

Comment:
I suppose it could have gone into `Write.Tx.Balance` too 🤔

Reply:
> I suppose it could have gone into `Write.Tx.Balance` too thinking

Definitely! We might need to revise our function and module structure at a certain point.